cognite/client/data_classes/data_modeling/query.py

```diff
@@ -201,6 +209,18 @@ def dump(self, camel_case: bool = False) -> dict[str, Any]:
             nodes["from"] = self.from_
         if self.filter:
             nodes["filter"] = self.filter.dump()
+        if self.through:
+            if len(self.through) != 3:
+                raise ValueError(f"`through` must be on the form (space, view/version, property), was {self.through}")
+            nodes["through"] = {
+                "view": {
+                    "type": "view",
+                    "space": self.through[0],
+                    "externalId": self.through[1].split("/")[0],
```

Comment:
Silly question, can external id or version have `/` in it?

Reply:
Version yes, externalId no.